uk_bin_collection/uk_bin_collection/councils/CotswoldDistrictCouncil.py

```
@@ -4,240 +4,138 @@
+                        )
+                        if container_type and raw_date:
                             data["bins"].append({
                                 "type": container_type,
-                                "collectionDate": collection_date
+                                "collectionDate": self._parse_date(raw_date, current_year),
+                            })
+                    except Exception:
+                        continue
+            else:
+                body_text = driver.find_element(By.TAG_NAME, "body").text
+                lines = [l.strip() for l in body_text.split("\n") if l.strip()]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)